.gitattributes

```diff
@@ -1,3 +1,4 @@
 docs/* linguist-generated
 __snapshots__/* linguist-generated
```

Comment:
I think we need to expand this, as well - perhaps `**/__snapshots__/*` (since I'm seeing some of the snapshots below)

Reply:
good call